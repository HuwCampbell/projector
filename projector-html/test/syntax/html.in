{ foo (<b>bar</b>) <span>baz</span> <h1>quux</hq> }