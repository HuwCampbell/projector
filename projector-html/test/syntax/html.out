Template
  ()
  Nothing
  (TENode
     ()
     (THtml
        ()
        [ TExprNode
            ()
            (TEApp
               ()
               (TEApp
                  ()
                  (TEVar () TId { unTId = "foo" })
                  (TENode
                     ()
                     (THtml
                        ()
                        [ TElement
                            ()
                            (TTag () "b")
                            []
                            (THtml () [ TPlain () TPlainText { unTPlainText = "bar" } ])
                        ])))
               (TENode
                  ()
                  (THtml
                     ()
                     [ TElement
                         ()
                         (TTag () "span")
                         []
                         (THtml () [ TPlain () TPlainText { unTPlainText = "baz" } ])
                     , TWhiteSpace () 1
                     , TElement
                         ()
                         (TTag () "h1")
                         []
                         (THtml () [ TPlain () TPlainText { unTPlainText = "quux" } ])
                     , TWhiteSpace () 1
                     ])))
        ]))